src/components/courselayout.tsx

```diff
@@ -0,0 +1,20 @@
+interface CourseLayoutProps {
+  title: string;
+  description: string;
+}
+
+const CourseLayout: React.FC<CourseLayoutProps> = ({ title, description }) => {
+  return (
+    <div className="mx-auto flex w-full max-w-[calc(100%-100px)] flex-col items-center rounded-[20px] bg-course-layout-background/80 p-8 text-center font-Malayalam font-normal">
```

Comment:
the tailwind classes that are being used here are overcomplicated please ask for help tomorrow